recipes/nasm/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+import os
+from conans import ConanFile, tools
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch",
+
+    def test(self):
+        if not tools.cross_building(self, skip_x64_x86=True):
+            self.run("nasm --version", run_environment=True)
+            asm_file = os.path.join(self.source_folder, "hello_linux.asm")
+            out_file = os.path.join(self.build_folder, "hello_linux.o")
+            bin_file = os.path.join(self.build_folder, "hello_linux")
+            self.run("nasm -felf64 {asm_file} -o {out_file}", run_environment=True)
```

Comment:
```suggestion
            self.run(f"nasm -felf64 {asm_file} -o {out_file}", run_environment=True)
```
the f for the format string is missing, that's why it's failing the v1 test_package

Reply:
Thanks, I thought I checked that but must've been looking at test_package's copy.